app/boards/arm/corneish_zen/Kconfig.defconfig

```diff
@@ -0,0 +1,119 @@
+#
+# Copyright (c) 2021 Darryl deHaan
+# SPDX-License-Identifier: MIT
+#
+
+if BOARD_CORNEISH_ZEN_V2_LEFT
+
+	config ZMK_KEYBOARD_NAME
+	default "Corne-ish Zen"
+
+	config ZMK_SPLIT_ROLE_CENTRAL
+	default y
+
+endif # BOARD_CORNEISH_ZEN_V2_LEFT
+
+if BOARD_CORNEISH_ZEN_V2_RIGHT
+
+	config ZMK_KEYBOARD_NAME
+	default "Corne-ish Zen rt"
+
+endif # BOARD_CORNEISH_ZEN_V2_RIGHT
+
+
+if BOARD_CORNEISH_ZEN_V2_LEFT || BOARD_CORNEISH_ZEN_V2_RIGHT
+
+config BOARD
+	default "corneish_zen"
+
+config ZMK_SPLIT
+	default y
+
+#config I2C
+#	bool
+#	default y
+
+#config I2C_0
+#    bool
+#    default y
+#    depends on I2C
+
+#menuconfig SENSOR
+#    bool "Sensor Drivers"
+
+#config BQ274XX
+#    bool "BQ274xx Fuel Gauge"
+#    depends on I2C && SENSOR
+
+#config SENSOR_CAN_FETCH_ALL
+#	bool "Can the fuel gauge fetch all channels at once"
+#	default y
+
+config SPI
+    bool
+    default y
+
+config BT_CTLR
+	default BT
+
+config ZMK_BLE
+	default y
+
+config ZMK_USB
+	default y
+
+if USB
+
+config USB_NRFX
+	default y
+
+config USB_DEVICE_STACK
+	default y
+
+endif # USB
+
+config ZMK_DISPLAY
+	select LVGL_USE_CONT
+	select LVGL_FONT_MONTSERRAT_26
+	select LVGL_FONT_MONTSERRAT_20
+	select LVGL_FONT_MONTSERRAT_16
+	select LVGL_USE_LABEL
+	select LVGL_USE_IMG
+
+choice ZMK_DISPLAY_STATUS_SCREEN
+	default ZMK_DISPLAY_STATUS_SCREEN_CUSTOM
+endchoice
+
+if ZMK_DISPLAY
+
+config SPI
+	default y
```

Comment:
Already defaulted earlier... If SPI only needed w/ displays, remove the other default earlier in the file.